src/index.js

```diff
@@ -1,13 +1,126 @@
-export default function BaseballGame() {
-  this.play = function (computerInputNumbers, userInputNumbers) {
-    return "결과 값 String";
-  };
-}
-
-// export default class BaseballGame {
-//   play(computerInputNumbers, userInputNumbers) {
-//     return "결과 값 String";
-//   }
-// }
-
-new BaseballGame();
+export default class BaseballGame { 
+
+    constructor() {
+      this.computerNum = this.randomNumGenerator();
+      this.strike = 0;
+      this.ball = 0;
+      this.result = '';
+      console.log(this.computerNum);
+    }
+  
+    randomNumGenerator() {
+      let first = Math.floor(Math.random() * 9 + 1);
+      let second = Math.floor(Math.random() * 9 + 1);
+      while (first == second) {
+        second = Math.floor(Math.random() * 9 + 1);
+      }
+      let third = Math.floor(Math.random() * 9 + 1);
+      while (third == first || third == second) {
+        third = Math.floor(Math.random() * 9 + 1);
+      }
+      return (
+        third.toString() + second.toString() + first.toString()
+      );
+    }
```

Comment:
```suggestion
    randomNumGenerator() {
      let first = Math.floor(Math.random() * 9 + 1);
      let second = Math.floor(Math.random() * 9 + 1);
      while (first == second) {
        second = Math.floor(Math.random() * 9 + 1);
      }
      let third = Math.floor(Math.random() * 9 + 1);
      while (third == first || third == second) {
        third = Math.floor(Math.random() * 9 + 1);
      }
      return (
        third.toString() + second.toString() + first.toString()
      );
    }
```
randomNumGenerator()인데 반환값이 number가 아니고 string이여서 나중에 혼동의 소지가 있을것같아요!

`Math.floor(Math.random() * 9 + 1)`의 불필요한 중복이 많은것같아서 함수로 따로 뺴는것도 좋아보입니다.